recipes/pytest-executable/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pytest_executable" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: pytest-executable
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 316b7982bcdb923ad6b8453f54b348bf81328dcdcecad212034acdf592bd3a4d
+
+build:
+  number: 0
+  include_recipe: False
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
If you are restricting the version of `python` under `run`, please do the same under `host`

Reply:
OK.